Add a basic makefile (#162)
developmentseed/eoapi-k8s: README.md

  </a>
</p>

## What is eoAPI?

[https://eoapi.dev/](https://eoapi.dev/)

## Getting Started

Make sure you have [helm](https://helm.sh/docs/intro/install/) installed on your machine.

> Additionally, you will need a cluster to deploy the eoAPI helm chart. This can be on a cloud provider, like AWS, GCP, or any other that supports Kubernetes. You can also run a local cluster using minikube.i

Reviewer: ```suggestion
Additionally, you will need a cluster to deploy the eoAPI helm chart. This can be on a cloud provider, like AWS, GCP, or any other that supports Kubernetes. You can also run a local cluster using minikube.
```

### Local

For a local installation you can use a preinstalled [Minikube](https://minikube.sigs.k8s.io/), and simply execute the following command:

```bash
$ make minikube
```

> Once the deployment is done, you the url to access eoAPI will be printed at to your screen at the end.

Reviewer: ```suggestion
Once the deployment is done, the url to access eoAPI will be printed to your terminal.
```

### Cloud

If you don't have a k8s cluster set up on AWS or GCP then follow an IaC guide below that is relevant to you

> &#9432; The helm chart in this repo assumes your cluster has a few third-party add-ons and controllers installed. So
> it's in your best interest to read through the IaC guides to understand what those defaults are

* [AWS EKS Cluster Setup](./docs/aws-eks.md)
* [GCP GKE Cluster Setup](./docs/gcp-gke.md)

> Then simply execute the following command:

Reviewer: Maybe worth mentioning you'll need kubectl to be pointing to the cloud instance?

Author: Good point. I added a sentence.
